### ISSUE TITLE
Display with dark themes

### ISSUE THREAD

**Issue by maillardj** [erwanp/pytexit#36]
Hello!
Thank you for the package! 

In Dark Themes IPython the display output is hardly readable because the text colour is black. 

![image](https://user-images.githubusercontent.com/54948232/113731475-1cd9f000-96f9-11eb-86be-8f0cb31af6cb.png)

Does anyone know : 
- how to fix it? 
- how to automatically adapt to the user theme?

**Comment by sambiddledev** [comment]
Looking into this. Has something to do with the IPython.core.display. Can you share the steps to recreate this issue? Were you just using Ipython in the console? My console is not showing any LaTex output with IPython.

**Comment by erwanp** [comment]
I also have it here. I simply run it from the IPython 7.20.0 embedded in Spyder 4.2.1, and I get the LaTeX output.

**Comment by erwanp** [comment]
If you manage to find out why it's not happening in your own iPython, it would be nice if we can set it by default : it's very convenient to check the equations in your code

**Comment by sambiddledev** [comment]
Awesome. Thanks for sharing that. I'm going to look into it a bit more closely. Tried to reproduce using Jupyter notebook with a theme and it looked fine, so we may need to get it to ignore the theme in certain cases: [Jupyter Notebook Dark Mode Success](https://user-images.githubusercontent.com/17518619/114250831-d7fdc580-9953-11eb-8cef-29ad2b6160f6.png)

**Comment by sambiddledev** [comment]
@erwanp no equation displays when I use the IPython interpreter straight from my zsh terminal window on a mac. I'll look into that a bit more. Could be worthy of an issue as well?

**Comment by sambiddledev** [comment]
VS code seems to work without issue, also. So it appears to be interpreter specific. See example from Jupyter notebook embedded in vs code: 
![VScode Dark Mode](https://user-images.githubusercontent.com/17518619/114251185-f912e600-9954-11eb-86dc-7125cd78b08e.png)

**Comment by erwanp** [comment]
Nice!  So we have so far :
- jupyter notebook in VS code : ✔️ 
- IPython in mac terminal : ❌ (nothing appears) 
- IPython in Spyder, Windows : ✅ (but wrong background) 

@maillardj was it also in a Spyder IPython? Does it work in a IPython spawned from the console?

**Comment by maillardj** [comment]
Hello! Yes, it was in a spyder IPython. From gitbash console it doesn't print the equation!

**Comment by erwanp** [comment]
Extra search systematically pointed me to Jupyter Notebooks to render equations (which works well, as you showed @sambiddledev ) . IPython simply inherits the limitations of the console/terminal, which wasn't developed to show images.

I think we won't fix it !

**Comment by sambiddledev** [comment]
That makes sense. I wonder if some helper text just to indicate that if you want to see the equation we recommend viewing in xyz might be helpful. Just to provide a cue to the user

**Comment by erwanp** [comment]
Good idea, but the message shouldn't appear on a JupyterNotebook. Hard to know how which environment the line is executed.

Possible hack : what about subclassing the `IPython.core.display.Latex` name, which is shown here : 

![image](https://user-images.githubusercontent.com/16088743/114450607-8153ee00-9bd6-11eb-9c61-54ef94447f9f.png)


to something like `pytexit.use_a_jupyter_notebook_to_view_latex_rendering`
